### ISSUE TITLE
OTHER text in amount selector is cut off

### ISSUE THREAD

**Issue by blairlierman** [washingtonstateuniversity/WSUWP-Plugin-iDonate#46]
After manually zooming back in due to Issue #45, the OTHER amount selector has the text cut off on the right. 

![fund002](https://cloud.githubusercontent.com/assets/11821485/22219764/f558a08c-e160-11e6-91dd-ce7be6cfa607.png)

This was reproduced on an iPhone 6 Plus with iOS 10 using Safari.

**Comment by blairlierman** [comment]
@NateOwen The recent changes in #49 don't appear to have fixed this issue either. The padding seems to be a bit too large still.

I can reproduce it on:
* Safari using the Responsive Design Mode
* the iOS simulator using an iPhone 6 Plus, 7 and 7 Plus, all running iOS 10.2
* on a iPhone 6 Plus device running 10.2.

**Comment by NateOwen** [comment]
@blairlierman Here's what I see on my iphone 6 on ios 10.2.1

![image1](https://cloud.githubusercontent.com/assets/8118154/22305131/3a2e588e-e2ef-11e6-86aa-38cdd5c4c377.PNG)

**Comment by blairlierman** [comment]
@NateOwen It appears to be related to the `width: calc(XX% - 5px);` for the amount selection buttons. 
Using the iPhone 6 **Plus** in the Simulator, XX is set to 20%. When I set it above 21%, it looks better (actually responding) and when I set it to 33%, it looks like your screenshot.

When I try an iPhone 6 in the Simulator, it does look like your screenshot, so perhaps it is using the 33% value as well.
